src/lib/containers/discussion_forum/ForumTable.tsx

```diff
@@ -24,9 +32,11 @@ export const ForumTable: React.FC<ForumTableProps> = ({
   limit,
   filter,
 }) => {
+  const { accessToken } = useSynapseContext()
   const [sort, setSort] = useState<DiscussionThreadOrder>(
     DiscussionThreadOrder.PINNED_AND_LAST_ACTIVITY,
   )
+  const [subscribed, setSubscribed] = useState<Subscription>()
```

Comment:
This always assumes the user is not subscribed to start. Need to use https://rest-docs.synapse.org/rest/GET/subscription/id.html to see if the user is already subscribed or not.